db/migrate/20220909190221_drop_cockpit_role.rb

```diff
@@ -0,0 +1,16 @@
+class DropCockpitRole < ActiveRecord::Migration[6.1]
+  class SettingsChange < ActiveRecord::Base; end
+
+  def up
+    say_with_time("Deleting cockpit role configuration") do
+      SettingsChange.where("key = '/server/role' AND value LIKE '%ws_cockpit%'").select('id', 'value').each do |row|
+        if row.value == 'ws_cockpit'
+          row.delete
+        else
+          # if this string is in the middle of a string, add a comma, otherwise, no need
+          row.update(:value => row.value.sub(/(,?)ws_cockpit(,?)/) { ($1 == $2 && $1 == ',') ? ',' : ''})
```

Comment:
Is it simpler/easier to read with to just split.delete.join?

```suggestion
          row.update(:value => row.split(",").tap { |r| r.delete("ws_cockpit") }.join(",")))
```

Reply:
_technically_ it will update rows that don't change, but that's a minor penalty given the small number of rows in the table.

Reply:
I had this before, but the `delete` was a little ugly. I will put it back